tests/pass/0weak_memory_consistency.rs

```diff
@@ -286,6 +286,40 @@ fn test_iriw_sc_rlx() {
     assert!(c || d);
 }
 
+// Another test for C++20 SCfix.
+fn scfix() {
+    let x = static_atomic_bool(false);
+    let y = static_atomic_bool(false);
+
+    let thread1 = spawn(move || {
+        let a = x.load(Relaxed);
+        fence(SeqCst);
+        let b = y.load(Relaxed);
+        (a, b)
+    });
+
+    let thread2 = spawn(move || {
+        x.store(true, Relaxed);
+    });
+    let thread3 = spawn(move || {
+        x.store(true, Relaxed);
```

Comment:
This should've been `y.store(true, Relaxed)` right?

Reply:
Oh... yes. Good catch!